source/ConditionSet.cpp

```diff
@@ -179,18 +188,20 @@ ConditionSet &ConditionSet::operator=(const ConditionSet &other)
 	literal = other.literal;
 	conditionName = other.conditionName;
 	children = other.children;
+	conditions = other.conditions;
 
 	return *this;
 }
 
 
 
 // Load a set of conditions from the children of this node.
-void ConditionSet::Load(const DataNode &node)
+void ConditionSet::Load(const DataNode &node, const ConditionsStore *conditions)
 {
 	// The top-node is always an 'and' node, without the keyword.
 	expressionOperator = ExpressionOp::AND;
 	ParseBooleanChildren(node);
+	this->conditions = conditions;
```

Comment:
The conditions need to be set before we start parsing.
```suggestion
	this->conditions = conditions;
	// The top-node is always an 'and' node, without the keyword.
	expressionOperator = ExpressionOp::AND;
	ParseBooleanChildren(node);
```